doc/tikv/README.md

```diff
@@ -54,7 +54,15 @@ Read more:
 
 ## MVCC
 
-TODO
+TiKV supports storing multiple values for a database key. Suitably coded clients can use this mechanism to implement both optimistic and pessimistic transactions (Multi Versioned Concurrency Control). Whenever TiKV stores a row key, it concatenates the raw bytes of the key with a 64-bit timestamp value. Thus, the row key has the logical format key:timestamp1, key:timestamp2 and so on for its different versions. Helper functions exist to affix the timestamp to a given raw key, and to extract only the raw key from a versioned key.
```

Comment:
"Suitably coded clients" - I think something like "clients cooperating with TiKV" since, because TiKV presents clients an API based on transactions, not on MVCC.

"format key:timestamp1, key:timestamp2 and so on" - "format `key,timestamp1`, `key,timestamp2` and so on".

"Helper functions exist" - could you link to the module please?

Reply:
Have made the changes. Rewrote the intro para a bit.